src/components/index/Nav.tsx

```diff
@@ -0,0 +1,31 @@
+import Link from "next/link";
+
+import { HOME_NAVIGATION } from "../../utils/consts";
```

Comment:
We should change prefix to `INDEX_` as our view name, and in relation to comment above export it from somewhere closer to our landing